pom.xml

```diff
@@ -108,7 +108,7 @@
         <kafka.version>2.2.2</kafka.version>
         <kotlin.version>1.3.61</kotlin.version>
         <log4j.version>1.2.17</log4j.version>
-        <log4j2.slf4j.binding.version>2.13.2</log4j2.slf4j.binding.version>
+        <log4j2.slf4j.binding.version>2.15.0</log4j2.slf4j.binding.version>
```

Comment:
@kwart Maybe we could replace `log4j2.slf4j.binding.version` with `log4j2.version`?